nautobot/extras/models/models.py

```diff
@@ -102,7 +102,12 @@ class ConfigContext(BaseModel, ChangeLoggedModel, ConfigContextSchemaValidationM
     regions = models.ManyToManyField(to="dcim.Region", related_name="+", blank=True)
     sites = models.ManyToManyField(to="dcim.Site", related_name="+", blank=True)
     locations = models.ManyToManyField(to="dcim.Location", related_name="+", blank=True)
-    roles = models.ManyToManyField(to="dcim.DeviceRole", related_name="+", blank=True)
+    roles = models.ManyToManyField(
+        to="extras.Role",
+        related_name="+",
+        blank=True,
+        limit_choices_to=RoleModelsQuery(),
```

Comment:
Maybe I'm missing something - I don't see why `RoleModelsQuery` applies here. That returns a list of ContentTypes, not a list of Roles?